docs/i18n/ru/troubleshooting.md

```
@@ -16,3 +16,89 @@
+
+**Варианты исправления**:
+
+1. Переключиться на Brave:
+```toml
+[web_search]
+enabled = true
+provider = "brave"
+brave_api_key = "<SECRET>"
+```
+
+2. Переключиться на Exa:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)